src/test/components/CallTreeSidebar.test.js

```diff
@@ -165,7 +165,7 @@ describe('CallTreeSidebar', function() {
     expect(queryByText(/Implementation/)).toBe(null);
 
     dispatch(changeSelectedCallNode(2, [A, B, D]));
-    expect(getByText(/Implementation/)).not.toBe(null);
+    expect(getAllByText(/Implementation/).length).toBeGreaterThan(0);
```

Comment:
~~Just wondering if this is a needed change :)~~
Ah I see, `getByText` throws if there's more than 1 found element. Thanks for the change :)